### ISSUE TITLE
Repeated css on back card due to `{{FrontSide}}`

### ISSUE THREAD

**Issue by lucashttip** [lukesmurray/markdown-anki-decks#2]
Hi,
First of all, this package is great! Thanks for making it available!

I've noticed though that when I style the `.question` with a border it is duplicated (border gets thicker) on the back of the card. I'm not super familiar with html and css but I imagine it's because it uses `{{FrontSide}}` inside the `.question`  div on the back template, so it might nest the `.card` and `.question` divs inside it and duplicate the css? I suppose simply changing `{{FrontSide}}` to `{{Question}}` would fix it, but I'm know sure if this was intended of not.

**Comment by lukesmurray** [comment]
Happy to try and diagnose this issue. I somehow missed the notification for it otherwise I would've responded sooner. Could you post a code snippet containing markdown which reproduces the issue as well as any custom css you're using?

**Comment by lucashttip** [comment]
Sure! 
No problem!
So, you can test it on a super simple markdown as the following:
```
---
css: 'custom-styles.css'
---

# TestDeck

## Question
Answer
``` 
Here's the css I used:
```
.card {
  font-family: Helvetica;
  text-align: center;
  color: #fff;
  background-color: #282a36;
}

.question{
  background-color: #44475a;
  border: 2px solid #bd93f9;
  color: #f8f8f2;
  font-size: 15px;
  margin: 0 auto;
  width: 75%;
}


.answer {
  background-color: #44475a;
  border: 2px solid #50fa7b;
  color: #f8f8f2;
  font-size: 20px;
}
```
Front side of the card looks great but the back side presents the issue. 
Thanks for the attention!

**Comment by lukesmurray** [comment]
This should be fixed in [0.3.1](https://pypi.org/project/markdown-anki-decks/). Thanks for reporting the issue!

**Comment by lukesmurray** [comment]
Pushed another fix just now which should update all your existing cards if you change styling. Previously only new cards would get the new styling.
